### PR TITLE
Script to add `host` label to tools k8s resources

### PR DESCRIPTION
**Pull request by xoen**
This is to simplify unidling logic by leveraging labels.

### CODE REVIEW

**Comment by xoen** [comment]
**NOTE**: The bit that patch jobs is currently failing for some reason I don't get (yet!).

For example:

```bash
$ kubectl --namespace=user-alice patch job jupyter-lab-alice-jupyt --type='json' -p="[{\"op\": \"add\", \"path\": \"/spec/template/metadata/labels/host\", \"value\": \"alice-jupyter-lab.tools.example.com\"}]"

The Job "jupyter-lab-alice-jupyt" is invalid: spec.template: Invalid value: core.PodTemplateSpec{ [...]
```

Odd. Even trying to add the label manually by editing the job gives the same error 🤷‍♂️

File: scripts/set-tools-host-labels.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Add the `host` label to all RStudio/JupyterLab resources
+
+
+set -e
```
Comment (djsd123) [inline scripts/set-tools-host-labels.sh]: You're handling envvars not being set so won't suggest `set -u`.  Maybe `set -o pipefail` might be useful

**Comment by xoen** [comment]
Regarding helm, I didn't think too much about it but I think there are two main reasons why I went for the Bash script instead:
1. I don't want to mass upgrade all tools just to add `host` labels. As you know some users have "special sauce" images and mass upgrade using helm would bring all RStudio/JupyterLab to the latest versions in helm. This is not ideal IMHO.
2. Helm is quite peculiar and in this specific case I prefer to deal directly with `kubectl` instead, this means I don't end up with a error halfway a migration because of some Helm "design decision".

I hope it makes sense.

**Comment by xoen** [comment]
@djsd123 Thanks 🙏 

```
APP="jupyter-lab" ./scripts/set-tools-host-labels.sh
APP="rstudio" ./scripts/set-tools-host-labels.sh
```

**Pull request merged**